ocp_resources/dns.py

```diff
@@ -0,0 +1,9 @@
+from ocp_resources.resource import Resource
+
+
+class DNS(Resource):
```

Comment:
I believe we need to follow the contributing requirements:
https://github.com/RedHatQE/openshift-python-wrapper/blob/main/CONTRIBUTING.md#adding-a-new-module-resource

Reply:
iiuc this is a resource added by k8s (a user cannot add it).
please add a link to the api

Reply:
Ack.